src/eos-dropbox-app

```diff
@@ -198,10 +199,49 @@ class DropboxLauncher():
             logging.info("Not the main launcher instance. Exiting")
             self._mainloop.quit()
 
+    def _launch_dropbox(self):
+        self._launch_dropbox_daemon()
+        self._setup_config_monitor()
+
+    def _setup_config_monitor(self):
+        config = Gio.File.new_for_path(os.path.expanduser(os.path.expanduser(DROPBOX_CONFIG)))
+        self._config_monitor = config.monitor(Gio.FileMonitorFlags.NONE)
+        self._config_monitor.connect('changed', self._on_config_changed)
```

Comment:
My opinion is that this should be kept as a thin wrapper and we shouldn't assume that, because a side effect like this might look ok now, it will be ok in the future, as we don't control dropbox and this might hit us later on.

So, I strongly think you should cancel and/or disconnect from here once the job is done, and keep the wrapper as simple as possible.